### PR TITLE
Handy tech driver updates

### PR DESCRIPTION
**Pull request by FelixGruetzmacher**
### Summary of the issue:
Help Tech Activator Braille display was not supported.
### Description of user facing changes
Activator is now detected and can be used.
### Description of development approach
Modified driver and detection logic.
### Testing strategy:
Tested Braille with updated driver in developer scratchpad.
### Change log entries:
Help Tech Activator now supported.

### CODE REVIEW

**Comment by codeofdusk** [comment]
@FelixGruetzmacher A couple of asides if I may:

* The PR title refers to "Handy Tech", but the changelog and website are "Help Tech". Was the company renamed or merged? What's the relationship between Handy Tech and Help Tech?
* I'm **extremely** interested in the Activator whenever it comes out! Do you have any idea of a release date yet? (feel free to email me about this at my username at gmail dot com as not to clutter the PR comments).

File: source/brailleDisplayDrivers/handyTech.py [hunk 1]
```diff
@@ -103,6 +103,7 @@ def windowProc(self, hwnd: int, msg: int, wParam: int, lParam: int):
 MODEL_MODULAR_20 = b"\x80"
 MODEL_MODULAR_80 = b"\x88"
 MODEL_MODULAR_40 = b"\x89"
+MODEL_ACTIVATOR = b"\xA4"
```
Comment (codeofdusk) [inline source/brailleDisplayDrivers/handyTech.py:106]: I thought the Activator supported HID. Is there something in the HandyTech protocol that isn't supported by your HID implementation?

File: source/brailleDisplayDrivers/handyTech.py [hunk 2]
```diff
@@ -524,6 +525,20 @@ class Modular80(Modular):
 	numCells = 80
 
 
+class Activator(TimeSyncFirmnessMixin, AtcMixin, JoystickMixin, TripleActionKeysMixin, Model):
```
Comment (codeofdusk) [inline source/brailleDisplayDrivers/handyTech.py:528]: I thought the Activator supported HID. Is there something in the HandyTech protocol that isn't supported by your HID implementation?

Comment (FelixGruetzmacher) [reply source/brailleDisplayDrivers/handyTech.py:528]: The Activator supports both Braille HID and the Handy Tech Braille protocol. Several features, including time syncing with the computer, are not part of the HID standard and are therefore only supported in the Handy Tech protocol. Additionally, Braille HID for Activator may not be available via Bluetooth, in which case the Handy Tech Braille protocol must be used.
Handy Tech has been renamed to Help Tech, but the protocol pre-dates this.

File: source/brailleDisplayDrivers/handyTech.py [hunk 3]
```diff
@@ -524,6 +525,20 @@ class Modular80(Modular):
 	numCells = 80
 
 
+class Activator(TimeSyncFirmnessMixin, AtcMixin, JoystickMixin, TripleActionKeysMixin, Model):
+	deviceId = MODEL_ACTIVATOR
+	numCells = 40
+	genericName = name = 'Activator'
+
+	def _get_keys(self):
+		keys = super(Activator, self).keys
```
Comment (seanbudd) [inline source/brailleDisplayDrivers/handyTech.py:534]: I believe this notation is more standard for modern python

```suggestion
		keys = super().keys
```

File: source/brailleDisplayDrivers/handyTech.py [hunk 4]
```diff
@@ -524,6 +525,20 @@ class Modular80(Modular):
 	numCells = 80
 
 
+class Activator(TimeSyncFirmnessMixin, AtcMixin, JoystickMixin, TripleActionKeysMixin, Model):
+	deviceId = MODEL_ACTIVATOR
+	numCells = 40
+	genericName = name = 'Activator'
+
+	def _get_keys(self):
```
Comment (seanbudd) [inline source/brailleDisplayDrivers/handyTech.py:533]: Could this please get a type hint

```suggestion
	def _get_keys(self) -> Dict[int, str]:
```

**Review by seanbudd — APPROVED** [review]
Thanks @FelixGruetzmacher

**Pull request merged**